### ISSUE TITLE
Always adds the server name to the h1 tag

### ISSUE THREAD

**Issue by kghbln** [Hutchy68/pivot#76]
### Setup

- MediaWiki | 1.27.5 (b303614)02:26, 10 January 2019
- PHP | 7.0.33-0+deb9u1 (apache2handler)
- MariaDB | 10.1.37-MariaDB-0+deb9u1
- Pivot | 2.0.0 (15cce18) 20:35, 28 February 2018

### Issue

Originally reported on [mediawiki.org](https://www.mediawiki.org/wiki/Topic:Utjo03qg4u8yap9o):

> The H1 tag is a critical factor is SEO. This skin puts the Wiki name - not the page title - as the H1 tag on every page on the wiki. Please fix this error as this can have severe impact on SEO.

### Example

Source code of the [Grid page](https://pivot.wikiproject.net/wiki/Grid) on the demo wiki.

```html
<body class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Grid rootpage-Grid skin-pivot action-view">
<!-- START PIVOTTEMPLATE -->
		<div class="off-canvas-wrap docs-wrap" data-offcanvas="">
			<div class="inner-wrap">
								<nav class="tab-bar hide-for-print">
					<section id="left-nav-aside" class="left-small show-for-small">
						<a class="left-off-canvas-toggle"><span id="menu-user"><i class="fa fa-navicon fa-lg"></i></span></a>
					</section>
					
					<section id="middle-nav" class="middle tab-bar-section">
						<h1 class="title"><a href="/wiki/Main_Page">
					<span class="show-for-medium-up">MediaWiki Pivot Skin</span>
						<span class="show-for-small-only">
```

**Comment by Hutchy68** [comment]
Will fix as a I role in an update for Pivot using MW 1.32. Thanks @kghbln for reporting.

**Comment by Hutchy68** [comment]
Fixed in Release v2.1.0
